test/controllers/api/v2/content_view_versions_controller_test.rb

```diff
@@ -86,6 +86,18 @@ def test_index_with_content_view_by_version
       assert_equal expected_version.id, results['results'][0]['id']
     end
 
+    def test_index_with_organization_id
+      cvv = katello_content_view_versions(:library_default_version)
+
+      results = JSON.parse(get(:index, organization_id: cvv.organization.id).body)
+
+      assert_response :success
+      assert_template 'api/v2/content_view_versions/index'
+
+      assert_equal ['error', 'page', 'per_page', 'results', 'search', 'sort', 'subtotal', 'total'], results.keys.sort
+      assert results['results'].map { |r| r['id'] }.include?(cvv.id)
```

Comment:
@daviddavis Any idea how to ensure those versions outside the org aren't there? I definitely thought it would be a good idea. Should I just find one in the fixtures that belongs to a different organization and trust it never changes?